### PR TITLE
Add TryCloudflare

### PR DESCRIPTION
**Pull request by maple3142**
Although Cloudflare is already there, but I think this is a different service similar to ngrok, but only support HTTP (?).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -752,6 +752,7 @@ Table of Contents
 
 ## STUN, WebRTC, Web Socket Servers and Other Routers
 
+   * [TryCloudflare](https://developers.cloudflare.com/argo-tunnel/trycloudflare) — Expose local HTTP servers through Argo Tunnel to public.
```
Comment (ripienaar) [inline README.md:755]: I'd prefer we just update the existing one with additional information

Comment (maple3142) [reply README.md:755]: But the existing CloudFlare entries are about CDN, DNS and Workers, which is quiet different from this feature, or you want to add CloudFlare into major cloud and group them together?

**Comment by ripienaar** [comment]
While they arent a major cloud provider I do think they can be combined into 1 in the CDN section

**Comment by maple3142** [comment]
You mean merge other Cloudflare products such as workers.dev into a single thing under CDN section?

**Comment by ripienaar** [comment]
Yeah - workers is its own brand so we can leave that one alone, combining the rest is good - its not ideal I agree but the list is very long and we have a number of these cross purpose tools combined into one but some like CF remains

**Comment by ripienaar** [comment]
thanks!

**Pull request merged**